src/java/htsjdk/samtools/SAMRecordSetBuilder.java

```diff
@@ -346,6 +347,7 @@ public void addPair(final String name, final int contig, final int start1, final
         end2.setAlignmentStart(start2);
         end2.setReadNegativeStrandFlag(true);
         end2.setCigarString(readLength + "M");
+        end2.setAttribute(ReservedTagConstants.NM,0);
```

Comment:
OK. fixed it. look again?

On Fri, Apr 24, 2015 at 2:56 PM, Yossi Farjoun <farjoun@broadinstitute.org>
wrote:

> impossible. :-/
>
> On Fri, Apr 24, 2015 at 2:13 PM, Nils Homer <notifications@github.com>
> wrote:
>
>> In src/java/htsjdk/samtools/SAMRecordSetBuilder.java
>> <https://github.com/samtools/htsjdk/pull/223#discussion_r29072067>:
>>
>> > @@ -346,6 +347,7 @@ public void addPair(final String name, final int contig, final int start1, final
>> >          end2.setAlignmentStart(start2);
>> >          end2.setReadNegativeStrandFlag(true);
>> >          end2.setCigarString(readLength + "M");
>> > +        end2.setAttribute(ReservedTagConstants.NM,0);
>>
>> I don't think this is the only place where NM is not set (see the other
>> addPair methods). Do you want to add it to createReadNoFlag?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/samtools/htsjdk/pull/223/files#r29072067>.
>>
>
>